src/views/Home.vue

```diff
@@ -1,12 +1,15 @@
 <template>
 	<Content v-shortkey.once="['c']" app-name="mail" @shortkey.native="onNewMessage">
+		<AppDetailsToggle v-if="isMobile && showEnvelope"
```

Comment:
I'd probably rename that to `isThreadShown` to make it a bit clearer when this is visible.